### PR TITLE
Fix/multiple sg tests fix

### PR DESCRIPTION
**Pull request by sridevi-15**
#### Fixes #.

- [x] Ran `flake8`
- [x] Ran `run_repo_tests.sh`

#### Changes proposed in this pull request:

- fixed configuration of sync-gateway for topology specific tests
- fixed wrong blip exception handling on ios
- fixed multiple sgs for xamarin -ios

### CODE REVIEW

File: keywords/TestServeriOS.py [hunk 1]
```diff
@@ -94,12 +100,18 @@ def install_device(self):
...
+        if self.platform == "ios":
+            if self.debug_mode:
+                self.app_name = "{}-{}-Device-debug.app".format(self.app, self.version_build)
+            else:
+                self.app_name = "{}-{}-Device.app".format(self.app, self.version_build)
+        elif self.platform == "xamarin-ios":
+            self.app_name = "{}.app".format(self.app)
 
-        self.app_path = "{}/{}/{}".format(BINARY_DIR, self.app_dir, self.app_name)
+        if self.platform == "ios":
```
Comment (hrajput89) [inline keywords/TestServeriOS.py]: if and elif statement body are same

File: resources/sync_gateway_configs/listener_tests/multiple_sync_gateways_cc.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
             {{ autoimport }}
             {{ xattrs }}
             {{ no_conflicts }}
-            {{ revs_limit }}
+            "revs_limit": 25,
```
Comment (hrajput89) [inline resources/sync_gateway_configs/listener_tests/multiple_sync_gateways_cc.json]: why are we hardcoding values here?

File: resources/sync_gateway_configs/listener_tests/multiple_sync_gateways_cc.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
             {{ autoimport }}
             {{ xattrs }}
             {{ no_conflicts }}
-            {{ revs_limit }}
+            "revs_limit": 30,
```
Comment (hrajput89) [inline resources/sync_gateway_configs/listener_tests/multiple_sync_gateways_cc.json]: why are we hardcoding values here?

File: resources/sync_gateway_configs/listener_tests/multiple_sync_gateways_cc.json (hunk 2 above)
Comment (sridevi-15) [reply resources/sync_gateway_configs/listener_tests/multiple_sync_gateways_cc.json]: this files are required only for specific tests which needs specific revs limit. As it is multiple sgs, I have a problem of overriding cluster config with revision limit for temporary file as cluster configs temporary files gets override each other and both sync-gateways are having same revision limit at the end which I see issue and change it hard coded with different revision limit

File: resources/sync_gateway_configs/listener_tests/multiple_sync_gateways_cc.json (hunk 3 above)
Comment (sridevi-15) [reply resources/sync_gateway_configs/listener_tests/multiple_sync_gateways_cc.json]: same as above

**Pull request merged**